src/pages/page-my-profile.html

```diff
@@ -516,6 +541,25 @@
                 }
             }
             /**
+            * Toggles the visibility of the full Public Key
+            */
+            _togglePubKey() {
+                this.$.pubKey.classList.toggle('truncate');
```

Comment:
Should we use           

this.customStyle['--my-elem-visibility'] = 'visible';
this.updateStyles();

Instead of adding and removing classes?